docs/en_US/reference/experiment_config.rst

```diff
@@ -66,11 +65,11 @@ Mutually exclusive to `search space file`_.
 trial command
```

Comment:
for these fields, it is better to show users their real name, otherwise it is not straightforward to match.